src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_TypeChar.cs

```diff
@@ -263,13 +263,9 @@ private Document GetDocument()
 
         private CompletionHelper GetCompletionHelper()
         {
+            // Crash if we don't find a document, we're already in a bad state.
```

Comment:
Or is the GetHelper used more places that you want to crash, and this is the only one you commented on?